src/main/java/managers/FileReaderManager.java

```diff
@@ -0,0 +1,21 @@
+package managers;
+
+import dataProvider.ConfigFileReader;
+
+public class FileReaderManager {
+
+    private static FileReaderManager fileReaderManager = new FileReaderManager();
+
+    private static ConfigFileReader configFileReader;
+
+    private FileReaderManager(){
+    }
+
+    public static FileReaderManager getInstance() {
+        return fileReaderManager;
+    }
+
+    public ConfigFileReader getConfigFileReader() {
+        return (configFileReader == null) ? new ConfigFileReader() : configFileReader;
```

Comment:
break down by operation, **inline if** should look like:
**condition**
**?option1**
**:option2**

Reply:
Done